backend/server/src/main/java/com/issuetracker/dto/auth/UserInfoDto.java

```diff
@@ -4,16 +4,18 @@
 import com.fasterxml.jackson.annotation.JsonSetter;
 
 public class UserInfoDto {
-
+    private String login;
     private String name;
-
     private String avatarUrl;
+    
+    @JsonGetter("login")
+    public String getLogin() {
+        return login;
+    }
 
-    public UserInfoDto() {}
-
-    public UserInfoDto(String name, String avatarUrl) {
-        this.name = name;
-        this.avatarUrl = avatarUrl;
+    @JsonGetter("login")
+    public void setLogin(String login) {
+        this.login = login;
```

Comment:
```suggestion
    @JsonSetter("login")
    public void setLogin(String login) {
        this.login = login;
```

Setter로 변경해야할 것 같습니다!

Reply:
엇, 실수했군요. 알려주셔서 감사합니다.

Reply:
수정 완료했습니다.

73eed8b